src/org/testKitGen/ModesDictionary.java

```diff
@@ -98,7 +98,7 @@ private void parseInvalidSpec(Element modes) throws IOException {
 		ArrayList<String> specs = new ArrayList<String>();
 		int lineNum = 0;
 		BufferedReader reader = null;
-		if (arg.getSpec().toLowerCase().contains("zos")) {
+		if (arg.getSpec().toLowerCase().contains("zos") && !(arg.getJdkVersion().matches("[2-9][0-9]"))) {
```

Comment:
`arg.getJdkVersion()` is still used. Please ensure https://github.com/adoptium/TKG/pull/608#discussion_r1880445022 is addressed.